### DISCUSSION TITLE
New database idea

### DISCUSSION THREAD

**Discussion by liexes** [diamond-rip/Eden#318] [Ideas]
Add a posibility to connect mysql database (Please, need this)

**Comment by GoodestEnglish** [comment]
Not adding MySQL, here's reason: https://github.com/diamond-rip/Eden/wiki/Incompatibility
MySQL support will likely never come.

**Reply by liexes** [reply]
And the posibility to save the data in local server, and not in an database?

**Reply by GoodestEnglish** [reply]
No point to add local data saving. If you own a server you should have a database. If you want to test the plugin, you can always disable database saving to test anything.

**Reply by liexes** [reply]
But for the users what dont have databases? Only a server?

**Reply by liexes** [reply]
Or dont unknow of the theme of the free databases? (Sorry for my english, but i speak spanish)

**Reply by GoodestEnglish** [reply]
Well, every server should have a database. Servers who don't have a database should not be public. If your server is private, you can always turn off database feature.
